### PR TITLE
#6012:Move table from ComponentStyles  to a separate component called Table

### PR DESCRIPTION
**Pull request by ajaxbo360**
## Fixes Issue

 Closes issue : #6012 

## Changes proposed
 - Move table from ComponentStyles  to a separate component called Table

## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Screenshots
   -- creating table component :
Screen Shot 2023-04-12 at 12 28 25
 -- exporting  table component into ComponentStyle :
Screen Shot 2023-04-12 at 12 28 46

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/6021"></a>

### CODE REVIEW

**Review by ChinmayMhatre — CHANGES_REQUESTED** [review]

File: components/mdx/Table.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+
+import React from 'react';
```
[the remark this answers is not in the document]
Comment (ajaxbo360) [reply components/mdx/Table.js:2]: thanks for reviewing my code 👩‍💻

**Comment by eddiejaoude** [comment]
Please resolve conflicts

File: components/mdx/Table.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+export default function Table(props) {
+  return (
+    <div className="w-full overflow-auto">
+      <table {...props}></table>
+    </div>
+  );
+}
```
Comment (eddiejaoude) [inline components/mdx/Table.js:7]: Closing blank line is recommended by GitHub

```suggestion
}

```

File: components/mdx/ComponentStyle.js [hunk 3]
```diff
@@ -1,16 +1,13 @@
+import Table from "./Table";
```
[the remark this answers is not in the document]
Comment (ajaxbo360) [reply components/mdx/ComponentStyle.js]: Thank you for your support

**Review by eddiejaoude — APPROVED** [review]
Thank you 👍

**Pull request merged**